test/unit/localization/fi_test.rb

```diff
@@ -0,0 +1,118 @@
+# encoding: UTF-8
+
+require 'test_helper'
+require 'i18n'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.